src/utils/scrapers/library_scraper.py

```diff
@@ -4,11 +4,24 @@
 
 import xmltodict
 from bs4 import BeautifulSoup
+from dateutil.parser import parse
 from fastapi import HTTPException
 
 from src.utils import cached_request
 
 
+def process_timestamp(date):
+    if date is None:
+        return None
+    try:
+        date = parse(date)
+        timestamp = int(date.timestamp())
+    except ValueError:
+        # 如果日期不能被解析，返回 None
+        timestamp = None
+    return timestamp
```

Comment:
```suggestion
def process_timestamp(date):
    try:
        date = parse(date)
        timestamp = int(date.timestamp())
    except (ValueError, TypeError):
        # 如果日期不能被解析，返回 None
        timestamp = None
    return timestamp
```
或許可以改這樣把 None 排除（順便連其他不符合的型別一起弄掉）